tests/30rooms/06membership.pl

```diff
@@ -205,3 +206,129 @@
          );
       });
    };
+
+test "Can invite existing 3pid",
+   requires => [qw( user more_users test_http_server_uri_base do_request_json make_test_room await_http_request )],
+
+   do => sub {
+      my ( $user, $more_users, $test_http_server_uri_base, $do_request_json, $make_test_room, $await_http_request ) = @_;
+
+      my $invitee_email = "marmosets\@monkeyworld.org";
+      my $invitee_mxid = $more_users->[0]->user_id;
+      my $room_id;
+
+      Future->needs_all(
+         stub_is_lookup($invitee_email, $invitee_mxid, $await_http_request),
+
+         $make_test_room->([$user], visibility => "private")
+         ->then( sub {
+            ( $room_id ) = @_;
+            $do_request_json->(
+               method => "POST",
+               uri    => "/api/v1/rooms/$room_id/invite",
+
+               content => {
+                  id_server => (substr $test_http_server_uri_base, length("https://")),
```

Comment:
This can sit for now, but if it turns out we need that more often it might be best to provide another value into the test. env. containing just this, from `tests/01http-server.pl`